### ISSUE TITLE
Comments / editing on reviews

### ISSUE THREAD

**Issue by kajchang** [kajchang/studentsreview.me#22]

**Comment by Kirito139** [comment]
It is currently not possible to create a review for a teacher/course, correct? Is there a reason for this? Is this website still being maintained?

**Comment by kaijchang** [comment]
@Kirito139 Correct. I stopped hosting and maintaining this iteration of the project sometime in 2021 after I graduated. There is a static version up at https://studentsreview.github.io/fiat. I was going to find someone to take it over, but AFAIK Lowell had stopped doing Arena at that time so I didn't bother. Let me know if you're interested in reviving it, I'd be happy to provide any insights from my experiences with the project.

**Comment by Kirito139** [comment]
I'd love to revive it, but I currently do not have much free time to spend on it and so progress would be slow. Also I only have a little bit of programming experience in python (and practically none in other languages) and I haven't programmed in... too long so I am not sure how well I'd maintain it—though I am willing to learn. I noticed that the website at studentsreview.me shut down a couple months ago. Is there a reason it was but the GitHub page wasn't? Also, what's arena?

**Comment by kaijchang** [comment]
@Kirito139 I stopped paying for the studentsreview.me domain and just moved it to https://studentsreview.github.io/fiat, which is free. Arena was the name of Lowell's course selection system, where you could choose your own teachers/classes. Helping people with Arena was the main goal of the site.

**Comment by Kirito139** [comment]
Ah I see. Arena was tied to the mod system, yes? Sounds really nice, better than what we have now. However, I believe this project can still help with course selection, since students can review courses as well as teachers. Plus, it's nice to know what to expect when they tell you which teachers you have.
